### ISSUE TITLE
can you add blockstream to start9

### ISSUE THREAD

**Issue by redfang790** [Blockstream/satellite#98]
Would you mind putting blocksteam satellite on start9 here is the link https://docs.start9.com/latest/developer-docs/packaging

**Comment by groundstart** [comment]
Good idea. I would use this. I run two Start9s and have a Blockstream Satellite Pro Kit in storage.

**Comment by groundstart** [comment]
> > Good idea. I would use this. I run two Start9s and have a Blockstream Satellite Pro Kit in storage.
> 
> Yes and I want to use txtenna with it

Haven't heard that txtenna mentioned in a long time. I have 4 GoTenna Mesh devices.
